charts/kubernetes-policy-controller/templates/configmap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-kubernetes-matches
+  namespace: {{ .Release.Namespace }}
+data:
+  matches.rego: |-
+    package k8s
+    import data.kubernetes
+
+    matches[[kind, namespace, name, resource]] {
+      resource := kubernetes[kind][namespace][name].object
+    }
+
+    matches[[kind, namespace, name, resource]] {
+        resource := kubernetes[kind][namespace][name]
+    }
+---
+apiVersion: v1
```

Comment:
It's not a common practice to have multiple resources specified in a single manifest.